packages/driver/src/cy/commands/navigation.ts

```diff
@@ -1087,9 +1222,9 @@ export default (Commands, Cypress, cy, state, config) => {
             return
           }
 
-          // if it came from the user's onBeforeLoad or onLoad callback, it's
+          // if the err came from the user's onBeforeLoad/onLoad callback or is cross-origin, it's
           // not a network failure, and we should throw the original error
-          if (err.isCallbackError) {
+          if (err.isCallbackError || err.isCrossOrigin) {
             delete err.isCallbackError
```

Comment:
do we need to delete this as well?
```suggestion
            delete err.isCallbackError
            delete err.isCrossOrigin
```

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/21137/commits/43dfc2af97b71d44b9842e7647abf14eb1940f93